adapters/Unity/UnityAdapter/GADMAdapterUnitySingleton.m

```diff
@@ -97,19 +96,15 @@ - (void)requestRewardedAdWithDelegate:
 
   [self addAdapterDelegate:adapterDelegate];
 
-  if ([UnityAds isInitialized]) {
-    // Call metadata load API
-    NSString *uniqueEventId = [[NSUUID UUID] UUIDString];
-    UADSMetaData *loadMetaData = [[UADSMetaData alloc] initWithCategory:@"load"];
-    [loadMetaData set:uniqueEventId value:placementID];
-    [loadMetaData commit];
-
-    if ([UnityAds isReady:placementID]) {
-      [adapterDelegate unityAdsReady:placementID];
-    }
-  } else {
+  [UnityAds load:placementID];
```

Comment:
Is it safe to call `[UnityAds load:]` before
- Checking if UnityAds has been initialized?
- Checking if an ad is ready for a placement?

Reply:
@fritzhuie Any update on this?